src/InputField/README.md

```diff
@@ -18,6 +18,7 @@ Table below contains all types of the props available in InputField component.
 | help          | `React.Node`      |              | The help to display beneath the InputField.
 | label         | `string`          |              | The label for the InputField. [See Functional specs](#functional-specs)
 | inlineLabel   | `boolean`         |              | If true the label renders on the left side of input
+| inputRef      | `func`            |              | Prop for forwarding ref the of input. [See Functional specs](#functional-specs)
```

Comment:
should be just `ref` everybody know this prop in react

Reply:
@wEprous why inputRef ?

Reply:
It's outdated. Already changed 🙂